### ISSUE TITLE
Custom email template fails to compile

### ISSUE THREAD

**Issue by mmmewk** [aws-samples/amazon-cognito-passwordless-auth#152]
I followed the documentation here: https://github.com/aws-samples/amazon-cognito-passwordless-auth/blob/main/CUSTOMIZE-AUTH.md to setup custom email templates and ran into an error

```
SyntaxError: Cannot use import statement outside a module.
```

This was fixed by adding:

```
import * as lambdaNodeJs from "aws-cdk-lib/aws-lambda-nodejs";

createAuthChallenge: {
  entry: "functions/src/passwordless/createAuthChallenge.ts",
  bundling: {
    format: lambdaNodeJs.OutputFormat.ESM,
    banner: "import{createRequire}from 'module';const require=createRequire(import.meta.url);",
  },
},
```

Looks like it needs ESM to use the import syntax.

**Comment by ottokruse** [comment]
You're right, we do set it to [ESM in the lib](https://github.com/aws-samples/amazon-cognito-passwordless-auth/blob/7fa7b0b8edad3b3bee4f0024aa8ed7b3c7d96098/cdk/lib/cognito-passwordless.ts#L333), but if you override the `bundling` property, you should add it again as you did.

Thanks for reporting. You're probably not the first one to run into this!

Want to send a PR to add it to the code example in the docs?

**Comment by sparksoftltd** [comment]
I'm having the same issue and @mmmewk 's solution works fine, thanks!
